imagededup/methods/cnn.py

```diff
@@ -1,10 +1,11 @@
+import warnings
```

Comment:
Thanks for the isort pointer. There does not seem to be a specification about ordering `from ..` and `import ..`. For this codebase, we have stuck to using alphabetical sorting instead as it allows one to look at all the functionality imported from a given package in one glance regardless of the import form. Additionally, isort doesn't seem to create 3 different blank line separated groups of imports as mentioned in the PEP spec:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

So, it would be great if you could stick to the guidelines used in this codebase.

Reply:
Sure, I'll do that.

isort can do that btw :) I think something like this in `setup.cfg` should get about what you want:

``` setup.cfg
force_sort_within_sections = true    
known_first_party = imagededup   
```
If you're interested in this kind of tooling you could take a look at [black](https://github.com/psf/black) too.